### DISCUSSION TITLE
How to use next-sitemap in monorepo setup using nx workspace?

### DISCUSSION THREAD

**Discussion by Akshat424** [iamvishnusankar/next-sitemap#889] [General]
I am having a monorepo setup with nx workspace.
It doesn't have `postbuild` to build next-sitemap, though i figured out a similar way with `nx:run-commands`
The `dist` directory for nx apps will be `/dist/<app-name`, How to tell next-sitemap to use that folder?

**Comment by eduardonard** [comment]
Having same problem

**Comment by Akshat424** [comment]
I think it is not required now, NextJs now provides it out of the box. Same can be achieved with using `robots.ts` and `sitemap.ts`
Ref - [NextJs sitemap docs](https://nextjs.org/docs/app/api-reference/file-conventions/metadata/sitemap)
